01-main/packages/rcloneview

```diff
@@ -0,0 +1,14 @@
+DEFVER=1
+ARCHS_SUPPORTED="amd64 arm64"
+case ${HOST_ARCH} in
+    amd64) ARCH=x86_64 ;;
+    arm64) ARCH=aarch64 ;;
+esac
+get_website "https://rcloneview.com/src/download.html"
+if [ "${ACTION}" != "prettylist" ]; then
+    URL="$(sed -E -n "s/.*href=.(.*linux.*${ARCH}.*\.deb).*/\1/p" "${CACHE_FILE}")"
+    VERSION_PUBLISHED="$(cut -d'-' -f2 <<<${URL})"
+fi
```

Comment:
Given there is an EULA we should probably show it

```suggestion
fi
EULA="By downloading, installing, copying, saving on your computer, or otherwise using this software, you are consenting to be bound by all the terms and conditions of this EULA: \n - https://rcloneview.com/src/eula.html \nIf you do not agree to the terms and conditions of this agreement, you should not download, install and use the software.\n"

```